apps/api/docs/usage-guide.md

```diff
@@ -69,8 +71,8 @@ Replace `SERVER_API_KEY_VALUE` with the actual API key value you want to include
         "text": "This is a test notification",
         "html": "<b>This is a test notification</b>"
       },
-      "createdBy": "osmo_notify",
-      "updatedBy": "osmo_notify",
+      "createdBy": "osmoX",
```

Comment:
Update to `OsmoX`

Reply:
Addressed in https://github.com/OsmosysSoftware/osmo-x/pull/152/commits/ffca8627cb4ae9663e16c6d1288440193d1ada1b